runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -264,8 +264,11 @@ else if (!config.scoutOverlayAtBank())
 
 		if (client.getSetting(Setting.IN_RAID_PARTY) == -1)
 		{
-			overlay.setScoutOverlayShown(false);
-			raid = null;
+			if (!inRaidChambers || !config.keepScouterActiveDuringRaid())
```

Comment:
When a raid starts `IN_RAID_PARTY` is set to `-1`. AFAIK it doesn't change again after leaving at the end of a raid, so as you added the toggle currently looks to me like it would keep the overlay active forever if you have the scout overlay enabled at the bank.
I'm fairly sure you can  test this just by going into the lobby, starting the raid and then leaving it.

Reply:
As soon as the party dissolves, whether because you leave the clan chat or the raid ends, the scout overlay disappears. To be honest, I'm not entirely sure why, but it works. Tried leaving with Overlay at Bank both on and off.

Reply:
Oh right, it's because it checks it on every `VarbitChanged` that is fired.